test/framework/resource/k8s/node/wrapper.go

```diff
@@ -46,9 +46,9 @@ func GetNodeAndWaitTillCapacityPresent(manager Manager, os string, expectedResou
 	return observedNodeList
 }
 
-// VerifyCNINodeCount checks if the number of CNINodes is equal to number of nodes in the cluster, and verifies 1:1 mapping between CNINode and Node objects
+// VerifyCNINode checks if the number of CNINodes is equal to number of nodes in the cluster, and verifies 1:1 mapping between CNINode and Node objects
 // Returns nil if count and 1:1 mapping exists, else returns error
-func VerifyCNINodeCount(manager Manager) error {
+func VerifyCNINode(manager Manager) error {
```

Comment:
I am thinking if `isEqual` need a retry since before calling `VeirfyCNINode()` we just verify if nodelist has length as expected nodes count. There is still some chance that the controller could take some time to get all CNINodes created, especially this test is involved in a larger scale. I would suggest to use a dynamic retry times based on nodes scale if possible.